src/deepsparse/transformers/eval_downstream.py

```diff
@@ -109,6 +112,82 @@ def perplexity_eval(args, batch_size=16, dataset_name="openai_humaneval"):
     return perplexity_metrics
 
 
+def perplexity_wikitext_eval(args, batch_size=16, dataset_name=""):
```

Comment:
why are we creating a duplicate of `perplexity_eval` function? i do not see a reason why we should have a separate function, that mostly is just adding boilerplate code.

Reply:
Tried to reuse this func, swapping in wikitext, got bad results.